src/main/java/net/neoforged/moddevgradle/internal/CreateLaunchScriptTask.java

```diff
@@ -61,8 +62,8 @@ abstract class CreateLaunchScriptTask extends DefaultTask {
     /**
      * Set to the desired Java runtime classpath.
      */
-    @Classpath
     @InputFiles
+    @PathSensitive(PathSensitivity.ABSOLUTE)
```

Comment:
I like the idea of explicitly marking properties with `ABSOLUTE` (it's the default). What do you think? Maybe this is not great...